client/matching/client.go

```diff
@@ -45,10 +45,10 @@ import (
 var _ matchingservice.MatchingServiceClient = (*clientImpl)(nil)
 
 const (
-	// DefaultTimeout is the default timeout used to make calls
+	// DefaultTimeout is the max timeout for regular calls
 	DefaultTimeout = time.Minute * debug.TimeoutMultiplier
-	// DefaultLongPollTimeout is the long poll default timeout used to make calls
-	DefaultLongPollTimeout = time.Minute * 2 * debug.TimeoutMultiplier
+	// DefaultLongPollTimeout is the max timeout for long poll calls
+	DefaultLongPollTimeout = time.Minute * 5 * debug.TimeoutMultiplier
```

Comment:
I'm still learning the codebase, what are the implications of changing the default timeout? Are there concerns for other rpcs?

Reply:
This is sorta weird, it's just a max timeout imposed on all callers on the client side (in this case, frontend calling matching, since matching calling matching has its own shorter timeouts). The four calls that use this timeout already impose their own timeout server-side. And the SDK sets a timeout of its own (70s for Go I think). So for normal operation it doesn't really change anything, certainly not the length of regular long polls.

The one case where it would change behavior is if someone called a long poll endpoint with a longer timeout (e.g. with a modified sdk), then frontend would do its call using a 5m instead of 2m timeout. Matching would still normally return after ~60s, but if matching disappeared and there were no keepalives (I think they're off by default for internode), then frontend might take 5m instead of 2m to return deadline exceeded to the sdk.